internal/buildscripts/packaging/fpm/etc/otel/collector/splunk-support-bundle.sh

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env bash
+
+# Copyright Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#######################################
+# Globals
+#######################################
+CONFDIR="/etc/otel/collector" # Default configuration directory
+DIRECTORY= # Either passed as CLI parameter or later set to CONFDIR
+TMPDIR="/tmp/splunk-support-bundle-$(date +%s)" # Unique temporary directory for support bundle contents
```

Comment:
I wonder if it's preferable to use `mktemp` for staging and then produce the support tarball in the directory where the command is invoked by default w/ configurable `--output`.

Reply:
I thought about this but feared compatibility across systems (not POSIX to my knowledge). Given installer script relies on /tmp already this felt "safe", but open to change this. @jchengsfx thoughts here?